components/lwip/CMakeLists.txt

```diff
@@ -90,6 +90,32 @@ set(srcs
     "port/esp32/netif/dhcp_state.c"
     "port/esp32/netif/wlanif.c")
 
+if (CONFIG_LWIP_SNMP_SUPPORT)
```

Comment:
We should still support GNU make build system. Do we expect the related change to appear in `component.mk`?